Move control flow export tests to new tracer

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #163580
* #163260
* __->__ #163259
* #163258
* #163137
* #163136

Differential Revision: [D82732614](https://our.internmc.facebook.com/intern/diff/D82732614)

tugsbayasgalan committed:
Move control flow export tests to new tracer

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Move control flow export tests to new tracer"




[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Move control flow export tests to new tracer"


Differential Revision: [D82732614](https://our.internmc.facebook.com/intern/diff/D82732614)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D82732614).

avikchaudhuri reviewed (approved):
just works? I mean, didn't need any further changes?

tugsbayasgalan committed:
Update on "Move control flow export tests to new tracer"


Differential Revision: [D82732614](https://our.internmc.facebook.com/intern/diff/D82732614)

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Move control flow export tests to new tracer"


Differential Revision: [D82732614](https://our.internmc.facebook.com/intern/diff/D82732614)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D82732614).
